amp/TEMPLATE/ampTemplate/dashboard/dev/app/js/app/models/enabled-charts-collection.js

```diff
@@ -1,38 +1,40 @@
 var Backbone = require('backbone');
 var _ = require('underscore');
 
-var EnabledChartModel = Backbone.Model.extend({
-
-});
+var EnabledChartModel = Backbone.Model.extend({});
 
 var EnabledChartsCollection = Backbone.Collection.extend({
-	model : EnabledChartModel,
-	url : '/rest/common/fm',
-	fetchData : function() {
-		var params = {
-			"detail-modules" : [ "DASHBOARDS" ]
-		};
-		this.fetch({
-			type : 'POST',
-			async : false,
-			processData : false,
-			mimeType : 'application/json',
-			traditional : true,
-			headers : {
-				'Content-Type' : 'application/json',
-				'Cache-Control' : 'no-cache'
-			},
-			data : JSON.stringify(params), // This is necessary due to
-											// incompatibilities with Jersey
-											// when receiving the params.
-			error : function(collection, response) {
-				console.error('error loading charts.');
-			},
-			success : function(collection, response) {
-				// console.log(response);
-			}
-		});
-	}
+    model: EnabledChartModel,
+    url: '/rest/common/fm',
+    fetchData: function () {
+        var params = {
+            'enabled-modules': false,
+            'reporting-fields': false,
+            'detail-flat': true,
+            'full-enabled-paths': false,
+            'detail-modules': ['DASHBOARDS']
+        };
+        this.fetch({
+            type: 'POST',
+            async: false,
+            processData: false,
+            mimeType: 'application/json',
+            traditional: true,
+            headers: {
+                'Content-Type': 'application/json',
+                'Cache-Control': 'no-cache'
+            },
+            data: JSON.stringify(params), // This is necessary due to
+            // incompatibilities with Jersey
+            // when receiving the params.
+            error: function (collection, response) {
+                console.error('error loading charts.');

```

Comment:
can we report the error to the user?

Reply:
No, this function shouldnt throw errors and it would take time to implement.